### PR TITLE
Added training/certification pricing details, OpenFin Conformance

### PR DESCRIPTION
**Pull request by robmoffat**

### CODE REVIEW

File: website/data/community.json [hunk 1]
```diff
@@ -17,7 +17,7 @@
     "type": "examples-and-training",
     "badges": [
     ],
-    "description": "<p>This course is designed for developers who are looking to build interoperable applications using the FDC3 standard, as well as community participants who provide FDC3 technology and services.</p>"
+    "description": "<p>This course is designed for developers who are looking to build interoperable applications using the FDC3 standard, as well as community participants who provide FDC3 technology and services.</p><p>FCFP is priced at $299 but free seats are available to FINOS members.</p>"
```
Comment (openfin-johans) [inline website/data/community.json:20]: This should not be FCFP right - LFD237? Or should be just say "this course is priced..."

File: website/data/training.json [hunk 2]
```diff
@@ -17,7 +17,7 @@
     "type": "examples-and-training",
     "badges": [
     ],
-    "description": "<p>This course is designed for developers who are looking to build interoperable applications using the FDC3 standard, as well as community participants who provide FDC3 technology and services.</p>"
+    "description": "<p>This course is designed for developers who are looking to build interoperable applications using the FDC3 standard, as well as community participants who provide FDC3 technology and services.</p><p>FCFP is priced at $299 but free seats are available to FINOS members.</p>"
```
Comment (openfin-johans) [inline website/data/training.json:20]: As above

**Review by openfin-johans — CHANGES_REQUESTED** [review]
Looks good to me but some questions on how to refer to the course in the pricing paragraph.

File: website/data/training.json (hunk 2 above)
Comment (robmoffat) [reply website/data/training.json]: Good catch, @openfin-johans

**Review by openfin-johans — APPROVED** [review]
Ship it!

**Pull request merged**